### ISSUE TITLE
SIFT OVA missing sift-cli

### ISSUE THREAD

**Issue by salty4n6** [teamdfir/sift#518]
Hi,
I just downloaded the OVA from the SANS website and it appears that sift-cli is missing from the OVA.  Is that right?

![image](https://user-images.githubusercontent.com/81833785/113458687-b7c08a80-93d8-11eb-8198-853e7917e5a4.png)

Thanks,
Salty

**Comment by ekristen** [comment]
Indeed it is missing, I'll make sure we get it added back. You can simply download it from the sift-cli repository so that you can use it as normal.

**Comment by salty4n6** [comment]
Thanks.

**Comment by salty4n6** [comment]
Hi.
I downloaded the latest version of the OVA - Ubuntu 20.04 and the sift program is missing.  Everything else appears to be there and functional.

Is sift supposed to be there or are we okay just doing sudo apt update && sudo apt upgrade to keep the system up to date?
Thanks,
Salty

**Comment by ekristen** [comment]
Hey @salty4n6 great question, I don't recall if we installed the SIFT-CLI in the latest update. You can always download and install it on the existing VM, you will not break anything.

**Comment by pyhoff** [comment]
Issue is we can't sift-cli get it to install on the OVA. I even tried using the Salt-stack install still failed, many issues with pip

**Comment by ekristen** [comment]
@pyhoff please open a new issue and provide logs.

**Comment by ekristen** [comment]
Please also note, that once https://github.com/teamdfir/sift-saltstack/pull/80 is merged, SIFT CLI will be officially deprecated as of March 1, 2023, and is replaced with [cast](https://github.com/ekristen/cast)
